probe_scraper/parsers/third_party/histogram_tools.py

```diff
@@ -517,11 +518,70 @@ def add_counter(context):
 
     return histograms
 
+
+def to_camel_case(property_name):
```

Comment:
fly-by nit: is this code living somewhere in Gecko too? I can't seem to find in [parse_histograms.py](https://searchfox.org/mozilla-central/source/toolkit/components/telemetry/build_scripts/mozparsers/parse_histograms.py).

Note: `parse_histograms.py` was named `histogram_tools.py` before in m-c. While refactoring Telemetry code, we re-organized naming a bit. The file here is a bit old, but it's that one. The idea is that, in the future, the content of the `third_party` directory will be imported as a dependency from a python package (i.e. [mozparsers](https://pypi.org/project/mozparsers/)). For this reason, this repo-specific changes should probably live in [histograms.py](https://github.com/mozilla/probe-scraper/blob/master/probe_scraper/parsers/histograms.py).

But I defer any final decision to @fbertsch :)

Reply:
This will be on parse_histograms.py once https://bugzilla.mozilla.org/show_bug.cgi?id=1578661 reaches mozilla-central.

Reply:
Great, sorry for the noise then :-)